lib/src/main/java/graphql/nadel/Nadel.kt

```diff
@@ -43,18 +43,17 @@ import graphql.schema.idl.ScalarInfo.GRAPHQL_SPECIFICATION_SCALARS as graphQLSpe
 
 class Nadel private constructor(
     private val engine: NadelExecutionEngine,
-    internal val serviceExecutionFactory: ServiceExecutionFactory,
     val services: List<Service>,
     val engineSchema: GraphQLSchema,
     val querySchema: GraphQLSchema,
-    // make these all internal once we merge api/ and engine-nextgen/
-    val instrumentation: NadelInstrumentation,
-    val serviceExecutionHooks: ServiceExecutionHooks,
-    val preparsedDocumentProvider: PreparsedDocumentProvider,
-    val executionIdProvider: ExecutionIdProvider,
-    val overallWiringFactory: WiringFactory,
-    val underlyingWiringFactory: WiringFactory,
-    val schemaTransformationHook: SchemaTransformationHook,
+    internal val serviceExecutionFactory: ServiceExecutionFactory,
+    internal val instrumentation: NadelInstrumentation,
+    internal val serviceExecutionHooks: ServiceExecutionHooks,
+    internal val preparsedDocumentProvider: PreparsedDocumentProvider,
+    internal val executionIdProvider: ExecutionIdProvider,
+    internal val overallWiringFactory: WiringFactory,
+    internal val underlyingWiringFactory: WiringFactory,
+    internal val schemaTransformationHook: SchemaTransformationHook,
```

Comment:
We get benefits like this. Since it's the same module now, we can further restrict visibility for members shared between `api/` and `engine-nextgen/`